com.surfshark.Surfshark.metainfo.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>com.surfshark.Surfshark</id>
+
+    <name>Surfshark</name>
+    <summary>Connect to secure, private web!</summary>
+    <url type="homepage">https://surfshark.com/</url>
+    <url type="contact">https://support.surfshark.com/</url>
+
+    <metadata_license>CC-BY-SA-4.0</metadata_license>
+    <project_license>LicenseRef-proprietary=https://surfshark.com/terms-of-service</project_license>
```

Comment:
Does this allow redistribution? If not sources need to be extra-data.

Reply:
You are right, we would want to distribute the .deb application artifact ourselves. Refactored the manifest to utilise `extra-data`.